build.gradle

```diff
@@ -25,7 +25,7 @@ project.ext {
 apply plugin: 'kotlin'
 
 group 'com.y4kstudios'
-version '1.0.3'
+version '1.0.4'
```

Comment:
Would you revert the version change? I like to keep the version change commits free of code changes